tests/sentry/api/endpoints/relocations/test_retry.py

```diff
@@ -172,6 +169,7 @@ def test_good_with_superuser_when_feature_disabled(
         )
 
     @patch("sentry.tasks.relocation.uploading_complete.delay")
+    @override_options({"relocation.enabled": False, "relocation.daily-limit.small": 2})
```

Comment:
nit: we should keep the ordering of `@patch` and `@override_options` consistent between tests, otherwise future readers may wonder if the order is load-bearing, as I did when looking at this. Since afaict the order does not affect test behavior, I would suggest alphabetical ordering: `@override_options`, then `@patch`